grande/config

```diff
@@ -0,0 +1,5 @@
+SUITE=${SUITE:-$UBUNTU_VERSION}
+MIRROR=${MIRROR:-http://archive.ubuntu.com/ubuntu}
+FLAVOUR=${FLAVOUR:-minimal}
+TMPFS_SIZE=${TMPFS_SIZE:-1073741824}
+BOOTSTRAP_INCLUDE=${BOOTSTRAP_INCLUDE:-ca-certificates,dumb-init,less,locales,iputils-ping,iproute2,sudo}
```

Comment:
Do we need sudo in container? Noone is using sudo - it should be root or normal user without any sudo rights.